pyaiot/gateway/coap/application.py

```diff
@@ -78,7 +78,7 @@ def run(arguments=[]):
         return
 
     start_application(CoapGateway(keys, options=options),
-                      port=options.coap_port, close_client=True)
+                      port=None, close_client=True)
```

Comment:
But how do you use a custom port then ?

Reply:
It's still handled in the gateway code

https://github.com/pyaiot/pyaiot/blob/1533b69e828fc1fc3cdaa442d7225b49d131b7f3/pyaiot/gateway/coap/gateway.py#L132-L152